src/test/lib/ApiTest.cpp

```diff
@@ -2787,6 +2787,52 @@ void QuicTestGlobalParam()
                 nullptr));
     }
 
+    //
+    // QUIC_PARAM_GLOBAL_STATISTICS_V2_SIZES
+    //
+    {
+        TestScopeLogger LogScope0("QUIC_PARAM_GLOBAL_STATISTICS_V2_SIZES");
+        uint32_t Length = 0;
+        //
+        // First call: expect buffer too small
+        //
+        TEST_QUIC_STATUS(
+            QUIC_STATUS_BUFFER_TOO_SMALL,
+            MsQuic->GetParam(
+                nullptr,
+                QUIC_PARAM_GLOBAL_STATISTICS_V2_SIZES,
+                &Length,
+                nullptr));
+        TEST_TRUE(Length >= sizeof(uint32_t));
+
+        //
+        // Second call: retrieve the sizes
```

Comment:
IMO, the main risk of bug in this API is when we the size provided is smaller than the max (easy to overflow).
It'd be good to have a test case for a partial query then the full query.